MB artifacts for TTL feature and README.md template

pubudup opened the pull request:
This PR contains the new README.md file template and a Java client to test Message Broker Time to Live feature.

pubudup committed:
Adding README.md template for WSO2 QA artifacts

pubudup committed:
Adding README.md template for WSO2 QA artifacts

pubudup committed:
update the format

pubudup committed:
update the format

pubudup committed:
added testcase reference

pubudup committed:
added testcase reference

pubudup committed:
added testcase reference

pubudup committed:
added testcase reference

pubudup committed:
added testcase reference

pubudup committed:
added testcase reference

pubudup committed:
Created a simple java client to test MB Time to Live feature

pubudup committed:
Created a simple java client to test MB Time to Live feature

pubudup committed:
}Merge branch 'master' of https://github.com/pubudup/wso2-qa-artifacts

pubudup committed:
updated java classes with localhost

pubudup committed:
fixed a typo

pubudup committed:
made changes to the java client

nu1silva commented:
Please make the following changes:
1. There seems to be 2 README's in the given PR remove the unwanted README.md file
2. remove the commented lines in the QueueSendDefault.java class
3. the pom.xml needs to be updated with the latest version on the andes client and related jars

pubudup committed:
removed commented lines

pubudup committed:
updated pom file with repo and jar versions

pubudup commented:
1. One README.me file is the template which is at the root and the other is for MB artefacts.
2. Removed commented lines.
3. Updated the pom.xml as discussed.

dakshika reviewed (commented):
Please format and clean up the java files
